docs/application/dotnet/samples/system/overview.md

```diff
@@ -71,10 +71,17 @@ for TD:
 			<p>This sample application demonstrates how to retrieve information about the physical orientation of the device using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Sensor.OrientationSensor.html"  target="_blank"> Tizen Orientation Sensor API</a>.</p>
 			</td>
 		</tr>
+		<tr>
+			<td><img alt="" height="180" src="media/w55deviceapp.png" width="180"/></td>
+			<td>
+			<p><strong>(W) DeviceApp</strong> [In progress]</p>
+			<p>This sample application shows the list of device features, such as battery, display, haptic, IR, LED, and camera back flash. It also tests their basic operations.</p>
```

Comment:
Please add "the" before "camera back flash".
Rephrase as:
This sample application shows the list of device features, such as battery, display, haptic, IR, LED, and the camera back flash. It also tests their basic operations.

Reply:
fixed it.